src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/ReferenceShard.java

```diff
@@ -0,0 +1,92 @@
+package org.broadinstitute.hellbender.engine.dataflow.datasources;
+
+import com.google.cloud.dataflow.sdk.coders.*;
+import com.google.cloud.dataflow.sdk.values.KV;
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.Serializable;
+
+/**
+ * ReferenceShard is section of the reference genome that's used for sharding work for pairing things with
+ * the reference. The primary use case is pairing reads with the reference.
+ * This class designed to be simple a data-storage class with related static utilities.
+ */
+public class ReferenceShard implements Serializable {
```

Comment:
This sounds like exactly the same thing as an interval. Why make a whole new class?

Statics are implicitly transient, so despite the comment about "this value is likely to be tweaked" we won't be able to safely write a transform that sets the shard size, creates a bunch of shards, and ships them to workers. I was fixing the BAM Reader just earlier today and one of the ways to make it scale is to determine the shard size dynamically at runtime, so this isn't just a hypothetical scenario.

I would recommend making the shard size a field of ReferenceShard, so it can be shipped around without surprise. Well actually I would recommend using the interval class since it represents the same thing and it's the same size anyways (one string, two ints).